.gitignore

```diff
@@ -31,3 +31,4 @@ pom.xml.versionsBackup
 .vim
 shell.nix
 tags
+**/*.jar
```

Comment:
JARs are build in `target/` so this ignore is unneccessary and removing this change will make your PR build a lot faster too as it will then just build your add-on and not everything! :slightly_smiling_face: 

```suggestion
```

Reply:
Done.